serverless-http.js

```diff
@@ -16,6 +16,10 @@ module.exports = function(app, opts) {
   const options = Object.assign({}, defaultOptions, opts);
 
   return (evt, ctx, callback) => {
+    if (options.callbackWaitsForEmptyEventLoop != null) {
```

Comment:
I think I am OK with just setting callbackWaitsforEmptyEventLoop to false always. You're not the first person who has been bitten by this, and the aws-express library does this. Sound good?